src/doc/src/reference/registry-web-api.md

```diff
@@ -306,6 +306,8 @@ A successful response includes the JSON object:
 
 ```javascript
 {
+    // Currently unused.
+    "msg": "This field is currently ignored by cargo.",
     // Indicates the remove succeeded, always true.
     "ok": true
```

Comment:
Following what ehuss suggested, I may do some change like this:


```suggestion
    // Indicates the remove succeeded, always true.
    "ok": true,
    // A string to be displayed to the user. Currently ignored by cargo.
    "msg": "owners successfully removed",
```

Reply:
Thanks for the input @weihanglo, I have made the changes and created another commit, but I do not know how to "squash my previous commits", doing on some research online I think it has to do something with "git rebase -i HEAD~n" where n should be the number of commits I want to squash into one.

But my recent git log looks like this-
```git
commit cc6ca48b63dc6a331fef7b7d6801ce17624b330c (HEAD -> #12056, origin/#12056)
Author: Bhardwaj-Himanshu <himanshujay261201@gmail.com>
Date:   Tue May 2 17:00:10 2023 +0530

    Some more reviewed changes for #12056.

commit 83a1469c319b6950826ad689eb3bf39874abbc4f
Author: Bhardwaj-Himanshu <himanshujay261201@gmail.com>
Date:   Tue May 2 13:52:20 2023 +0530

    Some more reviewed changes.

commit eb016a6df6715bb5d9bbec30e25999e70135f21d
Author: Bhardwaj-Himanshu <himanshujay261201@gmail.com>
Date:   Tue May 2 09:12:14 2023 +0530

    Reviewing changes as suggested in #12056

commit 0c535fc329843c4492e27e51e203108b54948d7d
Author: Bhardwaj-Himanshu <himanshujay261201@gmail.com>
Date:   Mon May 1 15:51:00 2023 +0530

    Should fix #12056.

commit ee7f4c3b3d3142b1d7bc10ef3a81dbb57fc055ba (origin/master, origin/HEAD, master)
Merge: 393ce5b41 b0a31036a
``` 

so should I add n as 3 and squash uptil the should fix #12056 commit?
Also I am trying this out locally on other test repos, I'll update you if it works without breaking anything!

Reply:
You can also reset all commits and re-commit it again, like

```
git reset --soft 0c535fc329843c4492e27e51e203108b54948d7d~
git commit -m 'Put your commit message here!!'
```

Use at you own risk.

Reply:
Then would you try to elaborate on the `squash` thing as well, to which commit should I squash?
Let's say I went back till 
- `commit ee7f4c3b3d3142b1d7bc10ef3a81dbb57fc055ba (origin/master, origin/HEAD, master)`
Which was made by bot
- But it would ask me to enter a message, which would create a new commit--> now when I add changes and make a new commit, both commits would be reflected in the changes, thus again adding 2 commits
  - one was for squash
  - one for the changes I made.
  
Thus looping me back on the multi-commit problem as well, do help me to resolve this,please!

Reply:
I don't really know what's going on with your machine.

If you can still find the branch of this PR (supposed to be `#12056`), you can

1. `git rebase --abort` to abort any previous rebase process.
2. `git reset --soft 0c535fc329843c4492e27e51e203108b54948d7d~` to reset all change from a commit before  0c535fc to the latest `HEAD` (current) into staged area.
3. `git commit -m'docs(registry): further specify owner-remove response'` to commit all changes in one commit
4. Then `git push`. You may need a force push with `--force` flag.

That's pretty much I can help for now. I am fine with merge it as-is :)